selectors.yml

```diff
@@ -184,3 +184,4266 @@ selectors:
+  8080808:
+    selector: 9044690156803099000
+    name: hokum
+  8794598:
+    selector: 3586950710777261600
+    name: hapchain
+  8888881:
+    selector: 7917262869722524000
+    name: quarix-testnet
+  8888888:
+    selector: 6468742140062220000
+    name: quarix
+  9322252:
+    selector: 5111203109553267000
+    name: xcap
+  9322253:
+    selector: 1529123784932564500
+    name: milvine
+  10067275:
+    selector: 6046219166406766000
+    name: plian-testnet-subchain-1
+  10101010:
+    selector: 5257098951723751000
+    name: soverun-mainnet
+  13068200:
+    selector: 2710868047971419600
+    name: coti-devnet
+  13371337:
+    selector: 8407997913912915000
+    name: pepchain-churchill
+  14288640:
+    selector: 9156987034660184000
+    name: anduschain-mainnet
+  16658437:
+    selector: 4960228519900718000
+    name: plian-testnet-main
+  17000920:
+    selector: 2879632549154365000
+    name: lambda-chain-testnet
+  18289463:
+    selector: 4114178857382327000
+    name: iolite
+  20180427:
+    selector: 4837189145333401000
+    name: stability-testnet
+  20180430:
+    selector: 6715128093199221000
+    name: smartmesh-mainnet
+  20181205:
+    selector: 9836073126363916000
+    name: quarkblockchain
+  20201022:
+    selector: 2051249019110326000
+    name: pego-network
+  20240324:
+    selector: 9686923291064832000
+    name: debank-sepolia-testnet
+  20482050:
+    selector: 8353951082740214000
+    name: hokum-testnet
+  22052002:
+    selector: 8528184855569132000
+    name: excelon-mainnet
+  27082017:
+    selector: 6752100751412897000
+    name: excoincial-chain-volta-testnet
+  27082022:
+    selector: 3703612276715058000
+    name: excoincial-chain-mainnet
+  28122024:
+    selector: 34731565802694764
+    name: ancient8-testnet
+  28945486:
+    selector: 5537762436579996000
+    name: auxilium-network-mainnet
+  29032022:
+    selector: 1256721375160818700
+    name: flachain-mainnet
+  31415926:
+    selector: 7060342227814389000
+    name: filecoin-local-testnet
+  35855456:
+    selector: 2047444139476444200
+    name: joys-digital-mainnet
+  37084624:
+    selector: 9659708103228396000
+    name: skale-nebula-hub-testnet
+  39916801:
+    selector: 4223478656447337000
+    name: kingdom-chain
+  43214913:
+    selector: 4324803305215419000
+    name: maistestsubnet
+  61717561:
+    selector: 5289868739243171000
+    name: aquachain
+  65010000:
+    selector: 4888170459303254000
+    name: autonity-bakerloo-thames-testnet
+  65010001:
+    selector: 4580382774751115000
+    name: autonity-bakerloo-barada-testnet
+  65010002:
+    selector: 510586525502394500
+    name: autonity-bakerloo-sumida-testnet
+  65100000:
+    selector: 7688769979994437000
+    name: autonity-piccadilly-thames-testnet
+  65100001:
+    selector: 9204657237950320000
+    name: autonity-piccadilly-barada-testnet
+  65100002:
+    selector: 4186165651132696000
+    name: autonity-piccadilly-sumida-testnet
```

Comment:
I think most (all?) instances of composed blockchain names separate them by `_`, and `-` separates the fields, i.e. chain family to net-type (testnet|mainnet)